Mathlib/FieldTheory/Galois/Basic.lean

```diff
@@ -548,6 +548,69 @@ instance (priority := 100) IsAlgClosure.isGalois (k K : Type*) [Field k] [Field
 
 end IsAlgClosure
 
+
+section restrictRestrictAlgEquivMapHom
+
+namespace IntermediateField
+
+variable {F E : Type*} [Field F] [Field E] [Algebra F E] (K L : IntermediateField F E) [Normal F K]
+
+/--
+The map from the `Gal(E/L)` to `Gal(K/F)` where `E/L/F` and `E/K/F` are two towers of
+extensions induced by the restriction to `K`. Note that we do require `K/F` to be normal but not
+`E/L`. If this is the case (and everything is finite dimensional) and `K ∩ L = F` then this
+map is surjective, see `IntermediateField.restrictRestrictMapHom_surjective`.
+This map is injective if the compositum of `K` and `L` is `E`,
+see `IntermediateField.restrictRestrictAlgEquivMapHom_injective`.
+-/
+noncomputable def restrictRestrictAlgEquivMapHom :
```

Comment:
Is it possible to generalize this definition from `IntermediateField` to `IsScalarTower`. Also, is it work generalizing to `IsGaloisGroup` or would that overly complicate things?

Reply:
Hum, what kind of statement do you have in mind?

Reply:
Given `IsScalarTower`s `E/L/F` and `E/K/F` satisfying `IsGaloisGroup G E L` and `IsGaloisGroup H K F`, define a homomorphism `G →* H`.

Reply:
Ah, I see. Is it at least possible to generalize to `IsScalarTower`?